cv_bridge/src/cv_bridge.cpp

```diff
@@ -72,6 +78,22 @@ static int depthStrToInt(const std::string depth) {
 
 int getCvType(const std::string& encoding)
 {
+
+#ifndef USE_BOOST_REGEX
+  std::regex FormatterFormatGroupPattern("\\$\\{([a-zA-Z]+)[}]", std::regex::egrep);
```

Comment:
You can drop both versions of `FormatterFormatGroupPattern`. And my comment about the `[}]`. Not needed in your case.